### PR TITLE
Nested exposures

### PR DESCRIPTION
**Pull request by Bugagazavr**
https://github.com/ruby-grape/grape-swagger-entity/issues/18

This feature implements a nested exposures parsing

screen shot 2017-02-26 at 22 31 20
screen shot 2017-02-26 at 22 31 30

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -1,43 +1,44 @@
...
+# Offense count: 2
 Metrics/PerceivedComplexity:
-  Max: 16
+  Max: 15
 
 # Offense count: 2
 Style/Documentation:
   Exclude:
     - 'spec/**/*'
+    - 'test/**/*'
```
Comment (LeFnord) [inline .rubocop_todo.yml]: can be removed

File: spec/grape-swagger/entities/response_model_spec.rb [hunk 2]
```diff
@@ -96,6 +96,23 @@ class Tag < Grape::Entity
           expose :name, documentation: { type: 'string', desc: 'Name' }
         end
 
+        class Nested < Grape::Entity
+          expose :nested, documentation: { type: 'Object', desc: 'Nested entity' } do
```
Comment (LeFnord) [inline spec/grape-swagger/entities/response_model_spec.rb]: please change one `object` to `Hash`, so others can see, this would be used synonym

**Comment by Bugagazavr** [comment]
@LeFnord done

**Comment by Bugagazavr** [comment]
@LeFnord i want release 0.2.0, but nested parsing it's a feature for `grape-entity` >= 0.5 versions, so maybe to drop support for 0.4?

**Comment by LeFnord** [comment]
@Bugagazavr go on … grape-entity support < 0.5 is always dropped

**Pull request merged**